app/src/main/java/practicum/getfitla_v3/NutritionItemFormat.java

```diff
@@ -28,16 +34,32 @@ public String getTitle() {
         return title;
     }
 
-    public String getShortdesc() {
-        return shortdesc;
+    public String getPrepTime() {
```

Comment:
Do you need all of these methods? Can't you just do nutritionObject.preptime if you want the preptime of the object?